src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -388,7 +388,7 @@ export class ExposureNotificationService {
   }
 
   private async processPendingExposureSummary() {
-    const summary = await this.exposureNotification.getPendingExposureSummary();
+    const summary = await this.exposureNotification.getPendingExposureSummary().catch(() => undefined);
```

Comment:
It's unnecessary to try catch PendingExposureSummary. We can return undefined and the method will return early anyway.